### PR TITLE
Autotests: # 3464 - Ignore chiral flag

### PR DESCRIPTION
**Pull request by MartaWilliams**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [x] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

File: ketcher-autotests/tests/Settings/Stereochemistry/ignore-chiral-flag.spec.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Verify absence "Enhanced Stereochemistry" group Label Behavior with Copy/Paste', async ({
+    page,
+  }) => {
+    // Test case: EPMLSOPKET-16920
+    const pointx = 204;
```
Comment (OlgaMazurina) [inline ketcher-autotests/tests/Settings/Stereochemistry/ignore-chiral-flag.spec.ts:60]: for constants it is right to use lowerCamelCase
so I suggest to use 'pointX' here

File: ketcher-autotests/tests/Settings/Stereochemistry/ignore-chiral-flag.spec.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    const pointy = 211;
```
Comment (OlgaMazurina) [inline ketcher-autotests/tests/Settings/Stereochemistry/ignore-chiral-flag.spec.ts:61]:
```suggestion
    const pointY = 211;
```

**Review by OlgaMazurina — APPROVED** [review]

**Review by Zhirnoff — APPROVED** [review]

**Comment by Zhirnoff** [comment]
Take a look at failing tests.
2023-11-01_13h15_54

**Review by Zhirnoff — COMMENTED** [review]
Take a look at failing tests please

**Review by Zhirnoff — CHANGES_REQUESTED** [review]
Take a look at failing tests.

File: ketcher-autotests/tests/Settings/Stereochemistry/ignore-chiral-flag.spec.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+async function applyIgnoreChiralFlag(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  await page.getByText('Stereochemistry', { exact: true }).click();
+  const deltaX = 0;
+  const deltaY = 60;
+  const anyX = 638;
+  const anyY = 524;
+  await page.mouse.move(anyX, anyY);
+  await page.mouse.wheel(deltaX, deltaY);
```
Comment (StarlaStarla) [inline ketcher-autotests/tests/Settings/Stereochemistry/ignore-chiral-flag.spec.ts:40]: here we could also use the common function to scroll right?

**Review by StarlaStarla — APPROVED** [review]

**Review by Zhirnoff — APPROVED** [review]

**Pull request merged**